pkg/sql/logictest/testdata/logic_test/sequences

```diff
@@ -350,11 +350,11 @@ SELECT nextval('spécial')
 statement ok
 CREATE TABLE kv (k bytes primary key, v bytes)
 
-statement error pgcode 42809 "kv" is not a sequence
+statement error pq: nextval\(\): "test.public.kv" is not a sequence
```

Comment:
can we keep the pgcode? i tend to remove `pq:` from the message as well (its regex)

Reply:
Done